pulse/uix/vtk/vtkMeshClicker.py

```diff
@@ -284,22 +412,24 @@ def pickPoints(self, x0, y0, x1, y1, tolerance=10):
         picker = vtk.vtkAreaPicker()
         extractor = vtk.vtkExtractSelectedFrustum()
         renderer = self.__rendererMesh._renderer
-        picker.AreaPick(x0,y0,x1,y1,renderer)
+        picker.AreaPick(x0, y0, x1, y1, renderer)
         extractor.SetFrustum(picker.GetFrustum())
 
         nodeBounds = self.__rendererMesh.nodesBounds
         camPos = renderer.GetActiveCamera().GetPosition()
-        distanceFromCamera = lambda key: distanceBoundsToPoint(camPos, nodeBounds[key])
-        pickedPoints = {key for key, bound in nodeBounds.items() if extractor.OverallBoundsTest(bound)}
-        
+        def distanceFromCamera(key): return distanceBoundsToPoint(
+            camPos, nodeBounds[key])
```

Comment:
Please add a newline after the declaration of the 
subfunction to avoid confusion.

Like this: 

```
def distanceFromCamera(key): 
    return distanceBoundsToPoint(camPos, nodeBounds[key])
```